src/components/common/BesideNavbar/BesideNavbar.jsx

```diff
@@ -0,0 +1,52 @@
+import { IconPlus } from 'components/UI/Icons/Icons';
+import { useUserData } from 'context/UserContext';
+import React, {useEffect, useState} from 'react';
+import {getAllUser} from 'utils/authService';
+import "./BesideNavbar.css";
+
+const BesideNavbar = () => {
+    const [userFollowArray, setUserFollowArray] = useState([]);
+    const { userData, setUserData } = useUserData();
+
+
+    useEffect(() => {
+        getAllUser(setUserFollowArray);
+    },[])
+    return (
+        <div className="BesideNavbar">
+            <div className='beside-navbar-container'>
+                <div className='beside-container-header'>
+                    <span className='fn-wg-700'>Who to Follow?</span>
+                    <span className='fn-wg-700 red-txt'>Show More</span>
+                </div>
+                <hr/>
+                <div className='beside-user-data-container'>
+                    {
+                    userFollowArray.map(user => (
+                        <div className='user-data-beside-section' key={user.userId}>
+                            <div className='user-data-beside-section-name'>
```

Comment:
use semantic tags, good for accessibility as well as SEO

Reply:
okay